<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная 1-2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>324</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lab1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_lab_1">
         <item>
          <property name="text">
           <string>GL_TRIANGLES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LINES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LINE_STRIP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_TRIANGLE_STRIP</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Lab2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Прозрачность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_transparency">
         <item>
          <property name="text">
           <string>GL_ALWAYS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LESS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_EQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_GREATER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_NOTEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_GEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_NEVER</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_transparency">
         <property name="maximum">
          <number>51</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Смешивание</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>sfactor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_mixing_1">
         <item>
          <property name="text">
           <string>GL_ZERO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA_SATURATE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>dfactor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_mixing_2">
         <item>
          <property name="text">
           <string>GL_ZERO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_STC_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_ALPHA</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Отсечение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_x">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_y">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_height">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_width">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="glScene" name="openGL_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glScene</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glscene.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
